webapp/src/DataFixtures/DefaultData/UserFixture.php

```diff
@@ -52,13 +53,19 @@ public function load(ObjectManager $manager)
                 throw new Exception(sprintf('Can not read admin password from %s', $adminPasswordFile));
             }
 
-            $adminuser = new User();
-            $adminuser
+            $adminUser = new User();
+            $adminUser
                 ->setUsername('admin')
                 ->setName('Administrator')
-                ->setPassword($this->passwordEncoder->encodePassword($adminuser, trim($adminpasswordContents)))
+                ->setPassword($this->passwordEncoder->encodePassword($adminUser, trim($adminpasswordContents)))
                 ->addUserRole($this->getReference(RoleFixture::ADMIN_REFERENCE));
-            $manager->persist($adminuser);
+            if (isset($_ENV['APP_ENV']) && $_ENV['APP_ENV'] == 'dev') {
```

Comment:
So when a "normal" install is put in debug mode, and the DB is reset this would apply but not when the DB is not recreated right?